src/disc/stk/Albany_AbstractSTKFieldContainer.hpp

```diff
@@ -25,6 +25,9 @@
 #include <stk_mesh/base/Field.hpp>
 #include <stk_mesh/base/FieldTraits.hpp>
 
+//#define HAVE_PROC_RANK_FIELD
+#undef HAVE_PROC_RANK_FIELD
```

Comment:
Why this undef?

Reply:
This will go away as soon as I get the parallel unit test for STKConnManager operational. I am trying to track down a memory issue that appears to be tied to the proc_rank field, using this construct.